apps/checkin/views.py

```diff
@@ -0,0 +1,130 @@
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from .models import CheckInGroup, CheckInEvent
+import qrcode
+from django.utils import timezone
+from .models import Applicant
+
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
+from rest_framework.views import APIView
+from django.http import HttpResponse
+
+static_path = "static/checkin/qr_codes/"
+
+
+class GetQRCode(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request):
+        # TODO: check that applicant was actually admitted
+        group = CheckInGroup(applicant=request.user)
```

Comment:
what if this already exists for the user?

seems weird that `GetQRCodeAdmin` has more safety checks when doing the same thing